### PR TITLE
New Component Selectable Card

### PR DESCRIPTION
**Pull request by thalytafabrine**
#### What is the purpose of this pull request?
As the title says.

#### What problem is this solving?
This Component is required for the new category view form in Admin Catalog.

#### How should this be manually tested?
[Running workspace](https://thalyta1--storecomponents.myvtex.com/admin/categories/create)

#### Screenshots or example usage
video

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by emersonlaurentino** [comment]
could you put an example in docs?

File: react/components/FloatingActionBar/index.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ const FloatingActionBar: FC<PropTypes.InferProps<typeof propTypes>> = ({
   save: { label: saveLabel, ...saveProps },
   cancel: { label: cancelLabel, ...cancelProps },
 }) => (
-  <div className="styleguide__floating-action-bar shadow-active w-100 bg-base tr pv5 pr7 absolute fixed bottom-0 left-0 z-2">
+  <div className="styleguide__floating-action-bar shadow-active w-100 bg-base tr pv5 pr7 absolute fixed bottom-0 left-0 z-999">
```
Comment (kevinch) [inline react/components/FloatingActionBar/index.tsx]: Is this not going to conflict with the modal?

File: react/components/SelectableCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+  return (
+    <div
+      className={`ba br2 relative ${
+        selected ? 'b--action-primary z-999' : 'b--transparent'
+      } nh2`}
+      style={{ borderWidth: '4px' }}>
+      {selected ? (
+        <div className="absolute right--1 top--1 br-100 h2 w2 bg-action-primary z-999">
+          <div className="pa3">
```
Comment (kevinch) [inline react/components/SelectableCard/index.tsx]: Could this class be set in the parent element and the `div` removed?

File: react/components/SelectableCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        style={{
+          boxShadow: '0 3px 9px 0 rgba(61, 62, 64, 0.25)',
+          clipPath: `inset(-10px ${hasGroupRight ? '0px' : '-10px'} -10px ${
+            hasGroupLeft ? '0px' : '-10px'
+          })`,
+        }}
+        className={`vtex-card card w-100 b2 br2 bg-base c-on-base ${padding} ${
+          onClick ? 'pointer' : ''
+        }`}
+        onClick={onClick ? () => onClick() : null}>
```
Comment (kevinch) [inline react/components/SelectableCard/index.tsx]: Shouldn't `onClick` be required?

File: react/components/FloatingActionBar/index.tsx (hunk 1 above)
Comment (thalytafabrine) [reply react/components/FloatingActionBar/index.tsx]: I don't think so, the modal z-index is 10000

File: react/components/SelectableCard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      {selected ? (
+        <div className="absolute right--1 top--1 br-100 h2 w2 bg-action-primary z-999 pa3">
+          <CheckIcon color="white" />
+        </div>
+      ) : null}
+      <div
+        style={{
+          boxShadow: '0 3px 9px 0 rgba(61, 62, 64, 0.25)',
+          clipPath: `inset(-10px ${hasGroupRight ? '0px' : '-10px'} -10px ${
+            hasGroupLeft ? '0px' : '-10px'
```
Comment (Klynger) [inline react/components/SelectableCard/index.tsx]: Pixels units can be just as numbers to the style porp

File: react/components/SelectableCard/index.tsx [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+      style={{ borderWidth: '4px' }}>
+      {selected ? (
+        <div className="absolute right--1 top--1 br-100 h2 w2 bg-action-primary z-999 pa3">
+          <CheckIcon color="white" />
+        </div>
+      ) : null}
+      <div
+        style={{
+          boxShadow: '0 3px 9px 0 rgba(61, 62, 64, 0.25)',
+          clipPath: `inset(-10px ${hasGroupRight ? '0px' : '-10px'} -10px ${
```
Comment (analuizamtg) [inline react/components/SelectableCard/index.tsx]: why negative pixels?

Comment (thalytafabrine) [reply react/components/SelectableCard/index.tsx]: to adjust the shadow when there are cards on the right or left

File: react/components/SelectableCard/index.tsx [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+  hasGroupRight,
+}) => {
+  const padding = noPadding ? '' : 'pa6'
+
+  return (
+    <div
+      className={`ba br2 relative ${
+        selected ? 'b--action-primary z-999' : 'b--transparent'
+      } nh2`}
+      style={{ borderWidth: 4 }}>
```
Comment (matheusps) [inline react/components/SelectableCard/index.tsx]: Use the `bw2` class from tachyons

**Pull request merged**